ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,430 @@
+---
+eip: to-be-assign
+title: Expirable Mechanism for ERC20 Token
+description: An extension of the ERC20 standard that enables the creation of fungible tokens with configurable expiration features, allowing for time-sensitive use cases.
+author: sirawt (@MASDXI), ADISAKBOONMARK (@ADISAKBOONMARK)
+discussions-to: https://ethereum-magicians.org/t/erc-xxxx-expirable-mechanism-for-erc20-token/21655
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7818-expirable-mechanism-for-erc20-token/21655
```
Added assigned number.
Recommend editing the discussion topic to just be a link to this PR, rather than a copy and paste of the proposed ERC as it will diverge as changes are made.